### ISSUE TITLE
bob: Update to clarify ambiguity regarding shouted questions

### ISSUE THREAD

**Issue by kytrinyx** [exercism/python#1166]
**TL;DR**: the problem specification for the Bob exercise has been updated.
Consider updating the test suite for Bob to match.
If you decide not to update the exercise, consider overriding description.md.

-----

**Details**

The problem description for the Bob exercise lists four conditions:

- asking a question
- shouting
- remaining silent
- anything else

There's an ambiguity, however, for shouted questions: should they receive
the "asking" response or the "shouting" response?

In https://github.com/exercism/problem-specifications/pull/1025 this ambiguity
was resolved by adding an additional rule for shouted questions.

If this track uses exercise generators to update test suites based on the
canonical-data.json file from problem-specifications, then now would be a good
time to regenerate 'bob'. If not, then it will require a manual update to the
 test case with input "WHAT THE HELL WERE YOU THINKING?".

See the [most recent canonical-data.json file](https://github.com/exercism/problem-specifications/blob/6055eb4b103e06c18184287b2be48f3c870aadcf/exercises/bob/canonical-data.json#L53-L58)
for the exact changes.

Remember to regenerate the exercise README after updating the test suite:

    configlet generate . --only=bob --spec-path=<path to your local copy of the problem-specifications repository>

You can download the most recent configlet at https://github.com/exercism/configlet/releases/latest if you don't have it.

If, as track maintainers, you decide that you don't want to change the exercise,
then please consider copying [problem-specifications/exercises/bob/description.md](https://github.com/exercism/problem-specifications/blob/8793a059fe18618fffeb97630baf76eebd5bf9a8/exercises/bob/description.md) into this track, putting it in
exercises/bob/.meta/description.md so that we can run the configlet README generation
without having to worry about the bob README drifting from the implementation.

----

PS: If this is unclear, would you help me figure out how to clarify it? I'd like to submit
this issue across all of the tracks that have implemented Bob, but that have not yet updated
to include the new rule.
